### DISCUSSION TITLE
Why transaction_guard_t can not be moved?

### DISCUSSION THREAD

**Discussion by SergeyBakev** [fnc12/sqlite_orm#909] [Q&A] [answered]
Why transaction_guard_t can not be moved?

**Comment by fnc12** [comment]
Cause nobody asked for it. Do you need it?

**Comment by SergeyBakev** [comment, marked as answer]
Yes. Now after move transaction it automatically committed in moved dctor

**Reply by fnc12** [reply]
what is 'move transaction'?

**Reply by SergeyBakev** [reply]
I create custom wrapper for transaction_guard_t. Into ctor i pass it(by rvalue). Moved guard in dctor automaticaly call commit(see gotta_fire). But now i own it in my wrapper

**Reply by fnc12** [reply]
Ok. Let me add it. I'll try to do it today

**Reply by fnc12** [reply]
https://github.com/fnc12/sqlite_orm/pull/911

**Reply by fnc12** [reply]
merged. Please check `dev` branch

**Comment by SergeyBakev** [comment]
Why operator=(&&) was removed?

**Reply by fnc12** [reply]
Cause otherwise implicit definition will be applied which doesn't act the way you expect.
